src/main/core/Controller/ResourceController.php

```diff
@@ -124,6 +131,17 @@ public function openAction($id, $embedded = 0)
                 return new JsonResponse(['resource_not_found'], 500);
             }
 
+            $resource = $this->om->getRepository($resourceNode->getClass())->findOneBy(['resourceNode' => $resourceNode]);
+
+            $this->strictDispatcher->dispatch(
+                ResourceEvents::RESOURCE_OPEN,
```

Comment:
Don't need a new dispatch.
There already is a generic event for that : https://github.com/claroline/Claroline/blob/13.x/src/main/core/Manager/ResourceManager.php#L599

Reply:
Change done, thx ^^